### PR TITLE
[CORE-4004] Adds carrier reference data for malca-amit

### PR DESCRIPTION
**Pull request by SanjayAtFlow**
Jira : https://flowio.atlassian.net/browse/CORE-4004
Discovery Doc: https://docs.google.com/document/d/1gHbVJhtZ2-Ycn8sWNCwXDjXA7ikQGZM6pkENihwtFXY/edit#

### CODE REVIEW

**Review by mikeclimbrock — CHANGES_REQUESTED** [review]
Are corresponding changes also needed to `carriers.csv` (https://github.com/flowcommerce/json-reference/blob/main/data/original/carriers.csv)?

**Comment by SanjayAtFlow** [comment]
> Are corresponding changes also needed to `carriers.csv` (https://github.com/flowcommerce/json-reference/blob/main/data/original/carriers.csv)?

As Paolo mentioned https://docs.google.com/document/d/1gHbVJhtZ2-Ycn8sWNCwXDjXA7ikQGZM6pkENihwtFXY/edit?disco=AAAASaINk3U , we need to add this for new external client.

**Review by paololim — CHANGES_REQUESTED** [review]
you need to add `malca` as a carrier in carriers.csv. from there, you'll need to regenerate the json files

**Review by paololim — CHANGES_REQUESTED** [review]
You shouldn't hand-update the json files.

Also, you should run the following in order:
1. `go run reference.go cleanse`
2. `git add <all the files related to carrier and carrier-serivces>`
3. `go run reference.go final`
4. `git add <all the files related to carrier and carrier-serivces>`

**Comment by rahul-flow** [comment]
> You shouldn't hand-update the json files.
> 
> Also, you should run the following in order:
> 
>     1. `go run reference.go cleanse`
> 
>     2. `git add <all the files related to carrier and carrier-serivces>`
> 
>     3. `go run reference.go final`
> 
>     4. `git add <all the files related to carrier and carrier-serivces>`

i didn't, i ran `go run reference.go all` and reverted all unecessary updates(like for currency,country etc.).

**Comment by rahul-flow** [comment]
> @rahul-flow are you sure it ran properly? the following files also need to be updated - and implies that the `go run reference.go final` wasn't ran, or that `all` failed at some point.
> 
> ```
> data/final/carrier-services.json
> data/final/carriers.json
> ```

I tried re-running reference.go with `all` and the steps you gave above, it doesn't seem to generate the `data/final/..` files. The original csv entries are to be added in `carriers.csv` and `carrier-services.csv` only right?

**Comment by rahul-flow** [comment]
Needed to run the steps individually since that currency error was coming, thanks Paolo!

File: data/original/carriers.csv [hunk 1]
```diff
@@ -19,3 +19,4 @@ ups,UPS,https://www.ups.com/track?loc=en_US&tracknum=
 usps,USPS,https://track.aftership.com/usps/
 wn-direct,WN Direct,http://wndirect.com/tracking.php?type=TR&ref=
 ocs-worldwide,OCS Worldwide,https://www.ocsworldwide.co.uk/Tracking.aspx?cwb=
+malca,Malca Amit,https://tracking.malca-amit.com/?t=
```
Comment (paololim) [inline data/original/carriers.csv:22]: how about let's call this `malca-amit`?

File: data/original/carrier-services.csv [hunk 2]
```diff
@@ -45,6 +45,7 @@ ilg-express-tracked,Express Tracked,ilg
 ilg-global-courier,Global Courier,ilg
 ilg-international-courier,International Courier,ilg
 ilg-standard-tracked,Standard Tracked,ilg
+malca-amit,Malca Amit,malca
```
Comment (paololim) [inline data/original/carrier-services.csv:48]: there should be two services here:
- armored
- express

based on the integration docs in https://my.malca-amit.com/integration/api

Screen Shot 2022-01-25 at 2 57 20 PM

**Review by paololim — CHANGES_REQUESTED** [review]

File: data/original/carriers.csv (hunk 1 above)
Comment (rahul-flow) [reply data/original/carriers.csv:22]: sounds good

File: data/original/carrier-services.csv (hunk 2 above)
Comment (rahul-flow) [reply data/original/carrier-services.csv:48]: ack, thanks!

**Review by paololim — APPROVED** [review]
looks good now, thank you!

**Pull request merged**